Package.swift

```diff
@@ -16,11 +16,13 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
         .package(url: "https://github.com/apple/swift-log.git", from: "1.0.0"),
         .package(url: "https://github.com/vapor/sql-kit.git", from: "3.0.0-rc.1"),
+        .package(url: "https://github.com/apple/swift-crypto.git", from: "1.0.0"),
```

Comment:
This is a huge additional dependency to pull in, including at the very least yet another copy of BoringSSL alongside NIO's. I'll speak to alternatives below.